ada_2_fhir-r4/zibs2020/payload/0.0/nl-core-MedicationContraIndication.xsl

```diff
@@ -46,6 +46,7 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
         <xsl:param name="in" select="." as="element()?"/>
         <xsl:param name="logicalId" as="xs:string?"/>
         <xsl:param name="subject" as="element()?"/>
+        <xsl:param name="reporter" as="element()?" select="melder/zorgverlener"/>

```

Comment:
De default is dat de ingevulde ADA-reference gebruikt wordt, maar dit kan overruled worden door het aanroepende template.

Reply:
Lijkt me prima. Al ga ik dan wel overal default waardes ("select") toevoegen (dus om het vullen van de params optioneel te maken inderdaad)